packages/trpc/server/routers/viewer/eventTypes/_router.ts

```diff
@@ -126,32 +127,38 @@ export const eventTypesRouter = router({
 
   get,
 
-  update: eventOwnerProcedure.input(ZUpdateInputSchema).mutation(async ({ ctx, input }) => {
-    const { updateHandler } = await import("./update.handler");
+  update: createEventPbacProcedure("eventType.update", [MembershipRole.ADMIN, MembershipRole.OWNER])
```

Comment:
Using new factory for PBAC procedure (specific to events here)
 
 Combines old logic plus checking permissions and falling back